test/conformance/testdata/conformance.txt

```diff
@@ -9,6 +9,18 @@ test/e2e/apimachinery/garbage_collector.go: "should not delete dependents that h
 test/e2e/apimachinery/garbage_collector.go: "should not be blocked by dependency circle"
 test/e2e/apimachinery/namespace.go: "should ensure that all pods are removed when a namespace is deleted"
 test/e2e/apimachinery/namespace.go: "should ensure that all services are removed when a namespace is deleted"
+test/e2e/apimachinery/resource_quota.go: "should create a ResourceQuota and ensure its status is promptly calculated."
+test/e2e/apimachinery/resource_quota.go: "should create a ResourceQuota and capture the life of a service."
+test/e2e/apimachinery/resource_quota.go: "should create a ResourceQuota and capture the life of a secret."
+test/e2e/apimachinery/resource_quota.go: "should create a ResourceQuota and capture the life of a pod."
+test/e2e/apimachinery/resource_quota.go: "should create a ResourceQuota and capture the life of a configMap."
+test/e2e/apimachinery/resource_quota.go: "should create a ResourceQuota and capture the life of a replication controller."
+test/e2e/apimachinery/resource_quota.go: "should create a ResourceQuota and capture the life of a replica set."
+test/e2e/apimachinery/resource_quota.go: "should create a ResourceQuota and capture the life of a persistent volume claim."
```

Comment:
Please remove the PV and PVC tests. They aren't ready for conformance yet. We don't have any PV or PVC tests yet.

Reply:
Removed.